src/pypromice/tx/get_watsontx.py

```diff
@@ -98,13 +98,16 @@ def get_watsontx():
             name=None
             d=None
         
-        if name and 'Watson station' in name:
-            print(f'Watson station message, {d.strftime("%Y-%m-%d %H:%M:%S")}')
+        if name and 'Watson' in name or name and 'GIOS' in name:
```

Comment:
I think this might be easier to read.
```suggestion
        if name and ('Watson' in name or 'GIOS' in name):
```

Reply:
This looks much cleaner, thanks!